React/Base/RCTUIKit.h

```diff
@@ -270,6 +270,9 @@ void UIGraphicsEndImageContext(void);
 // semantically equivalent types
 //
 
+// UIAccessibility.h/NSAccessibility.h
+@compatibility_alias UIAccessibilityCustomAction NSAccessibilityCustomAction;
```

Comment:
@tom-un When we move `RCTUIKit` upstream at some point, do you think it makes sense to rename these symbols to be platform agnostic? E.g. in this case `RCTAccessibilityCustomAction`?